### ISSUE TITLE
[resolved] Invalid revision to go.etcd.io/etcd in go.mod

### ISSUE THREAD

**Issue by kaloyan-raev** [minio/minio#10031]
We are trying to update our dependency to MinIO to the latest release.

## Expected Behavior

The `github.com/minio/minio` package should install without an issue.

## Current Behavior

It fails with:

```
go: github.com/minio/minio@v0.0.0-20200712190922-60d91234b974 requires
	go.etcd.io/etcd/v3@v3.0.0-00010101000000-000000000000: invalid version: unknown revision 000000000000
```

## Possible Solution

There is a `replace` statement in go.mod file that seems to work around the issue for minio builds, but causes trouble for integrators. The revision used in the `replace` statement should be used directly in the go.mod file.

## Regression
Yes, PR https://github.com/minio/minio/pull/9987

## Your Environment

* Operating System and version (`uname -a`):
```
Linux t480 5.5.10-100.fc30.x86_64 #1 SMP Wed Mar 18 14:34:46 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by klauspost** [comment]
Should be fixed by #10032

**Comment by kaloyan-raev** [comment]
Thank you!
